containers/ecr-viewer/src/app/services/socialHistoryService.tsx

```diff
@@ -22,7 +28,7 @@ export const evaluateTravelHistoryTable = (
     mappings.patientTravelHistory,
   );
 
-  const columnInfo = [
+  const columns: TravelHistoryColumn[] = [
```

Comment:
```suggestion
  const columns: ColumnInfoInput[] = [
```

then delete the `TravelHistoryColumn`

Reply:
Do you think it's worth doing something here to enforce requiring `infoPath`? I think that's what I was mainly after.

Reply:
maybe do `Required<Pick<ColumnInfoInput, "fields...">>` to at least make it clear its a subset of that?

Reply:
Yep, I like it. Thanks!